MediaBrowser.Controller/Collections/CollectionModifiedEventArgs.cs

```diff
@@ -1,24 +1,10 @@
-using System;
+﻿using System;
```

Comment:
BOM changes?

Reply:
Not consciously. Looking at it through Github desktop, I think there was a space in front of using.

Reply:
GitHub also hates diffs for the first line of a file in my experience.

Reply:
So this might just be a github thing?

Reply:
I checked this PR out locally and confirmed a BOM has been added to these files (at least two two I checked). The charset in our editorconfig file is set to `utf-8` (not `utf-8-bom`), so I think we should stick to that

Reply:
Ok. So I have no idea about this. I just use Visual Studio and Github Desktop. Is there a setting I should change on either of those 2?

Reply:
Hmm I'm not sure, but I know that Visual Studio respects editorconfig files so it probably isn't that. Maybe you can look through the settings in GitHub desktop to see if there's anything obvious?

Reply:
Nothing, but GHD's settings are very sparse to begin with. I did edit one file in VSCode because of a merge conflict and AFAIK Code does not respect Editor Configs out of the box. Could that be it?

And most importantly: how do I go about fixing this?

Reply:
I'm not sure how you can change it in Visual Studio. If you're not opposed to installing Visual Studio Code there is a button you can click in the bottom right corner to change it:

![image](https://user-images.githubusercontent.com/2626103/80171821-61887c00-85b9-11ea-9a07-a275f26f96b3.png)

Reply:
I already have VS Code. I have just pushed a commit that should fix this. Can you verify?

Reply:
Yup, looks good for this file! Can you fix this for the other files with BOM changes as well?

Reply:
What other files? Is there some way to get a list?

Reply:
You can look at the [file diff](https://github.com/jellyfin/jellyfin/pull/2937/files) in GitHub and look for files that have a change highlighted on the first line, but none of the text is changed. For example: https://github.com/jellyfin/jellyfin/pull/2937/files#diff-3db3cc510c449c0cb4edb0bdb7c70bc6

Reply:
I fixed some files I detected that way, but I also found some with more changes. Is there a systematic and automatic way to correct this?

Reply:
There's ~40 files in this PR, the fastest way is probably to go through and check them all with VS Code, I'm not sure of any other way

Reply:
So I wrote a quick tool to scan for a bom marker in the entire repository and I found the following files also still have BOM markers, but I'm sure that I didn't touch a lot of those:

d:\source\jellyfin\benches\Jellyfin.Common.Benches\HexDecodeBenches.cs
d:\source\jellyfin\benches\Jellyfin.Common.Benches\HexEncodeBenches.cs
d:\source\jellyfin\benches\Jellyfin.Common.Benches\Program.cs
d:\source\jellyfin\Emby.Server.Implementations\HttpServer\WebSocketConnection.cs
d:\source\jellyfin\MediaBrowser.Providers\Movies\ImdbExternalId.cs
d:\source\jellyfin\MediaBrowser.Providers\Plugins\AudioDb\Plugin.cs
d:\source\jellyfin\MediaBrowser.Providers\Plugins\MusicBrainz\Plugin.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Music\MultiDiscAlbumTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Subtitles\SubtitleParserTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\AbsoluteEpisodeNumberTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\DailyEpisodeTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\EpisodeNumberTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\EpisodeNumberWithoutSeasonTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\EpisodeWithoutSeasonTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\MultiEpisodeTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\SeasonFolderTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\SeasonNumberTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\TV\SimpleEpisodeTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\BaseVideoTest.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\CleanDateTimeTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\CleanStringTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\ExtraTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\Format3DTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\MultiVersionTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\StackTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\StubTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\VideoListResolverTests.cs
d:\source\jellyfin\tests\Jellyfin.Naming.Tests\Video\VideoResolverTests.cs
d:\source\jellyfin\MediaBrowser.Providers\Plugins\AudioDb\Configuration\PluginConfiguration.cs
d:\source\jellyfin\MediaBrowser.Providers\Plugins\MusicBrainz\Configuration\PluginConfiguration.cs

Should I remove the BOM marker on all of them?

Reply:
If you'd like to it would be welcome but maybe in a separate pull request so we can get this one through faster :) ideally that one will get merged quickly to prevent merge conflicts as well, so a separate PR makes it easier.